tooling/compose-intellij-platform/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,10 +1,17 @@
 <idea-plugin>
     <id>org.jetbrains.compose.intellij.platform</id>
-    <name>Compose for IDE Base</name>
+    <!--suppress PluginXmlValidity ("plugin" is usually not approved in the IJ marketplace, but it's OK for US) -->
```

Comment:
Why this line?

Reply:
Otherwise the name will be underscored red by IDE

> "plugin" is usually not approved in the IJ marketplace, but it's OK for US

In general Intellij verifier prohibits any name with the word "plugin", but I talked to the marketplace team, that we can ignore that rule in this case, because the word plugin actually makes sense here